mongodb-aggregate-query-support-test/pom.xml

```diff
@@ -17,5 +17,11 @@
       <artifactId>de.flapdoodle.embed.mongo</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <version>6.14.3</version>
+      <scope>test</scope>
```

Comment:
This should not be in test scope.  why do you need this in the test module

Reply:
It's official website has this: https://testng.org/doc/maven.html